stepper.c

```diff
@@ -30,13 +30,84 @@
 #include "report.h"
 #include "magazine.h"
 
+#include "spi.h"
+
 // Some useful constants.
 #define DT_SEGMENT (1.0/(ACCELERATION_TICKS_PER_SECOND*60.0)) // min/segment
 #define REQ_MM_INCREMENT_SCALAR 1.25
 #define RAMP_ACCEL 0
 #define RAMP_CRUISE 1
 #define RAMP_DECEL 2
 
+//SPI drivers
+#define SPI_ADDRESS_MASK        0x7
+#define SPI_RW_BIT              7
+// For use with the SPI driver chip
+typedef enum {
+  XTABLE,
```

Comment:
Just for sanity's sake, set the first element of the enum to zero. I need to go back and check the standard, but I'm not sure that the first element in this list is guaranteed a value otherwise.